### PR TITLE
allow to be built when using cocoapods

### PR DESCRIPTION
**Pull request by bmichotte**
See #159

### CODE REVIEW

File: JNWCollectionView/JNWCollectionViewFramework.h [hunk 1]
```diff
@@ -21,7 +21,11 @@
 #import "JNWCollectionViewCell.h"
 #import "JNWCollectionViewReusableView.h"
 #import "NSIndexPath+JNWAdditions.h"
-#import "JNWScrollView.h"
+#if defined(COCOAPODS)
+#import <JNWScrollView/JNWScrollView.h>
+#else
+#import JNWScrollView.h
```
Comment (jwilling) [inline JNWCollectionView/JNWCollectionViewFramework.h]: This is going to need to be wrapped in quotations.

**Comment by jwilling** [comment]
Thanks for putting this together! This seems to be the easiest way to solve the problem without reworking dependencies. I just have one comment.

**Comment by bmichotte** [comment]
Corrected, sorry for the missing quotes

**Comment by jwilling** [comment]
Awesome, thanks again! :sparkles:

**Pull request merged**